### PR TITLE
Added Implementation for Fantasy Hockey

### PR DESCRIPTION
**Pull request by willfeltman**
Created classes and tested for Fantasy Hockey Implementation

### CODE REVIEW

**Comment by cwendt94** [comment]
Looks good, thanks for adding the initial implementation for Hockey! Would you be able to add some unit tests for some of the features? Also I think you will need to add `pandas` to `setup.py` under `install_requires`.

File: espn_api/hockey/constants.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    , 22: 'Vancouver Canucks'
+    , 23: 'Washington Capitals'
+    , 24: 'Arizona Coyotes'
+    , 25: 'Anaheim Ducks'
+    , 26: 'Florida Panthers'
+    , 27: 'Nashville Predators'
+    , 28: 'Winnipeg Jets'
+    , 29: 'Columbus Blue Jackets'
+    , 30: 'Minnesota Wild'
+    , 37: 'Vegas Golden Knights'
```
Comment (austinegri) [inline espn_api/hockey/constants.py:41]: Any reason VGK is 37 here?

**Comment by willfeltman** [comment]
@cwendt94 I will add some Unit Tests and edit the pull request!

@austinegri I'm not entirely sure. That 37 is the team ID for the Knights, I'm guessing 31-36 are used in their database for dead teams (like the Atlanta Thrashers).

**Comment by cwendt94** [comment]
I'm not sure why codecov didn't pick up all the new files but let me know if you want some help for tests. I currently don't have a fantasy hockey league so I would need a public league to pull mock data from.

**Comment by willfeltman** [comment]
Sorry for the multiple commits, still new to Git Hub and ended up uploading files through the browser. 

Added in some simple Unit tests using data from existing private league. I wasn't able to find a public league since hockey isn't in session and leagueIds don't appear to be sequentially generated.

**Comment by cwendt94** [comment]
No problem, thanks for adding some simple tests. I will try to look at it in the next couple days.

If you don't mind I might update some of the naming just to keep it in line with the other sports and I can create some more tests now that you added some mock data.

**Comment by austinegri** [comment]
@cwendt94 Can we merge this request?

I found a public league that can be used for testing: leagueId=77421173, year=2021

**Comment by cwendt94** [comment]
@austinegri Sure, I can merge. Thanks for finding a public league!

I really wanted to refactor a lot of the interfaces to follow how basketball and football are formatted and write the documentation. However I have been pretty busy lately.

I will merge this in and create a ticket for me to refactor hockey!

**Comment by cwendt94** [comment]
See #190 for refactor

**Pull request merged**